Source/FakeItEasy/Core/FakeScope.cs

```diff
@@ -13,19 +16,48 @@ namespace FakeItEasy.Core
     internal abstract class FakeScope
         : IFakeScope
     {
+        private const string CurrentScopeLogicalCallContextKey = "FakeItEasy.FakeScope.Current";
+
+        private static readonly RootScope root;
+        private static readonly ConcurrentDictionary<Guid, FakeScope> scopes;

```

Comment:
@TimLovellSmith, logical call context values should be serializable when crossing remote boundaries. (And they do cross remote boundaries during testing -- between the test runner AppDomain and the tested code AppDomain.)